asllib/libdir/stdlib.asl

```diff
@@ -34,6 +34,11 @@ end;
 
 // Log2
 
+func IsPowerOfTwo(x: integer) => boolean
+begin
+  return x > 0 && x == 2 ^ Log2 (x);
```

Comment:
The previous documentation of `Log2` says "// Calculate the logarithm base 2 of the input. Input must be a power of 2." so you probably shouldn't count on `Log2` to round down.

Reply:
Why do you need the special cases below 2047?

Reply:
I understand, but why do we need to optimize? If there's a benefit for herd then it's a good reason, but in terms of providing an example implementation, it's not necessary. I guess if we had an optimized version for ASLRef/herd and one for external consumption, we won't have to compromise.